index.html

```diff
@@ -204,5 +204,66 @@ <h3>About Me</h3>
         <li class="my_skills">Terminal</li>
       </ul>
     </section>
+    <section class="motivation">
+      <p>
+        I'm always<br />
+        interested in hearing <br />
+        about new projects,<br />
+        so if you'd like to chat please get in <br />
+        touch.
+      </p>
+    </section>
+    <section class="form">
+      <form action="https://formspree.io/f/xeqnejbr" method="post">
+        <div>
+          <input
+            type="text"
+            id="full_name"
+            placeholder="Full name"
+            name="Full_name"
+            maxlength="30"
+            required
+          />
+        </div>
+        <div>
+          <input type="email" id="email" placeholder="Email" name="Email" />
+        </div>
```

Comment:
- [ ] Good job so far 🔥. Please let's make sure you have the same placeholder text with the template. The template has a placeholder text `Email address` but yours has `Email`. Can you fix this so that your design matches that of your chosen template?
- [ ] What do you think about adding the attribute `required` to the email field so that the user has to add an email address before he/she can submit the form? This will help in passing the correct and required values to our form.